src/main/kotlin/gg/essential/essential-loader.gradle.kts

```diff
@@ -0,0 +1,68 @@
+package gg.essential
+
+import gg.essential.gradle.multiversion.Platform
+import gg.essential.gradle.util.RelocationTransform.Companion.registerRelocationAttribute
+
+
+plugins {
+    id("gg.essential.loom")
+}
+
+val platform = Platform.of(project)
+
+val essentialLoader: Configuration by configurations.creating
+
+when {
+    platform.isLegacyForge -> {
+        dependencies {
+            "implementation"(essentialLoader("gg.essential:loader-launchwrapper:1.1.3")!!)
+        }
+        tasks.named<Jar>("jar") {
+            dependsOn(essentialLoader)
+            manifest.attributes(mapOf(
+                "FMLCorePluginContainsMod" to "Yes, yes it does",
+                "TweakClass" to "gg.essential.loader.stage0.EssentialSetupTweaker"
+            ))
+            from({ zipTree(essentialLoader.singleFile) })
+        }
+    }
+
+    platform.isFabric -> {
+        dependencies {
+            "include"("modRuntimeOnly"("gg.essential:loader-fabric:1.0.0")!!)
+        }
+    }
+
+    platform.isModLauncher -> {
+        dependencies {
+            val relocatedPackage = findProperty("essential.loader.package")?.toString() ?: throw GradleException("""
+                A package for the Essential loader to be relocated to has not been set.
+                You need to set `essential.loader.package` in the project's `gradle.properties` file to a specific package where Essential's loader will be relocated to.
```

Comment:
Why "specific" package? We don't need any specific package, we mostly don't care which package you chose as long as it doesn't conflict with anything.
I believe "concrete" would technically be the word to use but that just sounds overly verbose (properties can only really be set to concrete packages in the first place; the opposite would imply something like wildcard support), so I'd just not use any extra adjective at all here.